bootstrap/bootstrap.md

```diff
@@ -12,10 +12,13 @@ About 4 hours 45 minutes
 - [UI/UX lesson](/ui-ux-design/ui-ux-design.md)
 
 ### Motivation
-Bootstrap is a popular and easy-to-use front-end framework first released by Twitter. Utilizing Bootstrap significantly reduces the amount of time spent styling your webpages, especially for mobile-development. 
-**Which companies use bootstrap?**
-- Intel
-- Twitter
+Bootstrap is a popular and easy-to-use front-end framework first released by Twitter. Utilizing Bootstrap significantly reduces the amount of time spent styling your webpages, especially for mobile-development.
+
+**Example of working websites that have used bootstrap**
+
+Below are the examples of websites that have used bootstrap:
+- [Fliplingo](https://www.fliplingo.com/)
```

Comment:
his link is not working for me, and it says it is not a secure website .  Could you pick another?